### ISSUE TITLE
Add basic speaker information

### ISSUE THREAD

**Issue by rmw** [WriteSpeakCode/2016.writespeakcode.com#18]
For schedule:
- Add talk name + link to bio

Speaker page with all speaker info

For now just use ActiveYAML.

**Comment by rmw** [comment]
@dalrayes Any chance you have some time to work on this tomorrow?  I may be able to get to it, but not sure.

I've started a [branch](https://github.com/WriteSpeakCode/2016.writespeakcode.com/tree/rmw-speakers), which uses the [ActiveHash gem](https://github.com/zilkey/active_hash) to store the speaker info similarly to [how we did it last year](https://github.com/WriteSpeakCode/writespeakcode.github.io/blob/dev/data/people.yml).

I added a rake task (rake wsc:csv_to_json), which takes a csv file that has speaker info in the format we collected in CFP (using the "Speaker Info" file in google drive folder "Conference/2016/Speaker Outreach/Speaker Info").  This reads the csv file and then sends into a JSON file.

Things to do:
- Right now the JSON file is kinda overwhelming -> maybe instead, we use ActiveHash [multiple files](https://github.com/zilkey/active_hash#multiple-files-per-model) and have each speaker has their own .json file.  This should be easier to fix/update and not accidentally overwrite.  Maybe?
- Photos needs to be cut to be square. (Or automate with rake task/imagemagick?)
- Change to be "People" instead of "Speaker"? (Similar to last year)
- Ensure bios are up to date based on ZenDesk
